internal/convert/database_user_test.go

```diff
@@ -62,43 +52,37 @@ func TestBuildAtlasRoles(t *testing.T) {
 				RoleName:     "something",
 				DatabaseName: "admin",
 			},
+		}},
+	}
+
+	for _, tc := range tests {
+		got := BuildAtlasRoles(tc.input)
+		if err := deep.Equal(tc.want, got); err != nil {
+			t.Fatalf("expected: %v, got: %v", tc.want, got)
 		}
-		if err := deep.Equal(r, expected); err != nil {
-			t.Error(err)
-		}
-	})
+	}
 }
 
 func TestBuildOMRoles(t *testing.T) {
-	t.Run("No database defaults to admin", func(t *testing.T) {
-		r := BuildOMRoles([]string{"admin"})
-		expected := []*opsmngr.Role{
+	type test struct {
+		input []string
+		want  []*opsmngr.Role
+	}
+
+	tests := []test{
+		{input: []string{"admin"}, want: []*opsmngr.Role{
```

Comment:
[super-nit] this and for this file in general can we better split these lines, right now is hard to read